StateMachine.js

```diff
@@ -6,10 +6,10 @@ const Sentry = require('@sentry/node');
 Sentry.init({ dsn: 'https://9a178f83cd78420c960331dbc22e81d6@o248765.ingest.sentry.io/3011310' });
 let moment = require('moment');
 
-class SessionState {
+class StateMachine {
 
-    constructor(location) {
-        this.location = location;
+    constructor(locationid) {
+        this.location = locationid;
```

Comment:
Could be nice to change the name of the `location` property to also be `locationid`.

Reply:
I was nervous about doing that because if I remember right it is used as location in a bunch of different places from this where I'm not sure what's going on and I was scared of breaking that somehow